src/cellularnoise.rs

```diff
@@ -0,0 +1,100 @@
+use rand::*;
+use std::io::*;
+
+byond_fn! { cnoise_generate(precentage,smoothing_iterations, birth_limit, death_limit) {
+    noise_gen(precentage, smoothing_iterations, birth_limit, death_limit).ok()
+} }
+
+fn noise_gen(prec_as_str : &str, smoothing_level_as_str : &str, birth_limit_as_str : &str, death_limit_as_str : &str)-> Result<String> {
+    let prec = prec_as_str.parse::<i32>().expect("parse failed");
+    let smoothing_level = smoothing_level_as_str.parse::<i32>().expect("parse failed");
+    let birth_limit = birth_limit_as_str.parse::<i32>().expect("parse failed");
+    let death_limit = death_limit_as_str.parse::<i32>().expect("parse failed");
```

Comment:
These 4 should be using `?` instead of `.expect` so they don't crash the whole server on failure